core/main/src/bootstrap/boot.rs

```diff
@@ -64,6 +65,7 @@ pub async fn boot(state: BootstrapState) -> RippleResponse {
     execute_step(LoadExtensionsStep, &bootstrap).await?;
     execute_step(StartExtnChannelsStep, &bootstrap).await?;
     execute_step(StartAppManagerStep, &bootstrap).await?;
+    execute_step(StartAppsUpdaterStep, &bootstrap).await?;
```

Comment:
We cleaned up the steps that are needed post-session availability to the context manager. The contents of this step should be added as a [method](https://github.com/rdkcentral/Ripple/blob/613d8b283f52dcce50c7b4bc0f29d3cfeca84664/core/main/src/processor/main_context_processor.rs#L128) call in Context manager